build/IceRpc.Version.props

```diff
@@ -5,7 +5,8 @@
         <!-- The Protobuf version used by IceRpc.Protobuf -->
         <ProtobufVersion>26.1</ProtobufVersion>
         <NuGetProtobufVersion>3.$(ProtobufVersion)</NuGetProtobufVersion>
-        <TargetFramework>net8.0</TargetFramework>
-        <NetRuntimeVersion>8.0.*</NetRuntimeVersion>
+        <!-- The target framework used by assemblies -->
+        <IceRpcTargetFramework>net8.0</IceRpcTargetFramework>
+        <IceRpcNetRuntimeVersion>8.0.*</IceRpcNetRuntimeVersion>
```

Comment:
I rename this variable for consistency

Reply:
What should we set it to when using net9.0? I tried 9.0.* and it didn't work; 8.0.* seems to work but it's odd.

Reply:
The problem is that the expression `9.0.*` doesn't include preview versions. It should be `9.0.*-preview*`. I will add a comment on how to set this.

Reply:
Added comment wit example usage for .NET 9